eventually/authentication/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+URL Patterns
```

Comment:
this module's docstring is not necessary because of urls.py is been ignored by pylintrc's field.

Reply:
Fixed.